File: iPhone/WonBridge/Model/ContactEntity.swift
//
//  ContactEntity.swift
//  WonBridge
//
//  Created by Tiia on 16/09/16.
//  Copyright © 2016 elitedev. All rights reserved.
//

import UIKit

class ContactEntity: NSObject {
    
    var contactName: String!
    var nickName: String!
    var photoURL: String!
    var isMember: Bool!
    
    override init() {
        
        contactName = ""
        nickName = ""
        photoURL = ""
        isMember = false
    }
    
    convenience init(contactName: String, nickName: String, photoURL: String, isMember: Bool) {
        
        self.init()
        
        self.contactName = contactName
        self.nickName = nickName
        self.photoURL = photoURL
        self.isMember = isMember
    }
}

File: Android/app/src/main/java/com/julyseven/wonbridge/model/ContactEntity.java
package com.julyseven.wonbridge.model;

/**
 * Created by JIS on 9/15/2016.
 */
public class ContactEntity {

    String _contactName = "";
    String _nickName = "";
    String _photoUrl = "";
    boolean _isMember = false;

    public ContactEntity(String contactname, String nickname, boolean isMember) {

        _contactName = contactname;
        _nickName = nickname;
        _isMember = isMember;
    }

    public String get_contactName() {
        return _contactName;
    }

    public void set_contactName(String _contactName) {
        this._contactName = _contactName;
    }

    public String get_nickName() {
        return _nickName;
    }

    public void set_nickName(String _nickName) {
        this._nickName = _nickName;
    }

    public boolean is_isMember() {
        return _isMember;
    }

    public void set_isMember(boolean _isMember) {
        this._isMember = _isMember;
    }

    public String get_photoUrl() {
        return _photoUrl;
    }

    public void set_photoUrl(String _photoUrl) {
        this._photoUrl = _photoUrl;
    }
}
